jwst/pipeline/calwebb_spec3.py

```diff
@@ -61,12 +61,13 @@ def process(self, input):
             The exposure or association of exposures to process
         """
         self.log.info('Starting calwebb_spec3 ...')
+        self.allowed_exptypes = ['science']

```

Comment:
For spec3, I believe `BACKGROUND` should also be included. @hbushouse ?

Reply:
Correct. We need to keep both `science` and `background` member types.

You know, to avoid a LOT of confusion, we really should've called this attribute `memtype` (*member* type), instead of `exptype`. Is it too late for a total redesign?!

Reply:
Allowed exposure types are now ['science','background']